src/documentation/system_administrators/advanced/quantum_safe_storage/quantum_safe_storage_deploy.md

```diff
@@ -0,0 +1,594 @@
+
+```sh
+for node in $METADATA_NODES; do
+  name=node${node}meta
+  tfcmd deploy zdb --mode user --node $node -n $name --size 1 --password $PASSWORD
+done
+```
+
+It's possible that some of your chosen nodes don't respond properly at deployment time and need to be replaced with other nodes. In that case, just replace the variable with the new node IDs like this:
+
+```sh
+# Example: Node 3 wasn't working, replace it with node 5
+METADATA_NODES="1 2 4 5"
+```
+
+Then you can just run the original deployment script loop again. Any zdb deployments that would be duplicated will be detected by `tfcmd` and skipped over.
+
+### Cancel Metadata Zdbs
+
+If you need to cancel the metadata zdb deployments, use this script:
+
+```sh
+for node in $METADATA_NODES; do
+  name=node${node}meta
+  tfcmd cancel $name
+done
+```
+
+### Write Metadata Zdbs Config
+
+Next we will write the configuration data from the deployed zdbs into the config file. The script as shown is for IPv6 connections. If you are using Yggdrasil, replace `.Zdbs[0].ips[0]` with `.Zdbs[0].ips[1]`
+
+```sh
+# Wait about five seconds before doing the next step to make sure data is available. Only needed if running immediately after the deployment step in a single script
+sleep 5
+
+# Write out the config sections for the metadata zdbs
+for node in $METADATA_NODES; do
+  name=node${node}meta
+  echo Fetching and writing config for $name
+  json=$(tfcmd get zdb $name 2>&1 | tail -n +3 | sed $'s/\e\[0m//')
+  ip=$(echo $json | jq .Zdbs[0].ips[-1] | tr -d \")
+  port=$(echo $json | jq .Zdbs[0].port)
+  namespace=$(echo $json | jq .Zdbs[0].namespace)
+  password=$(echo $json | jq .Zdbs[0].password)
+
+  echo \# Node $node >> $CONFIG
+  echo [[meta.config.back ends]] >> $CONFIG
+  echo address = \"\[$ip\]:$port\" >> $CONFIG
+  echo namespace = $namespace >> $CONFIG
+  echo password = $password >> $CONFIG
+  echo >> $CONFIG
+done
+```
+
+Once that has completed, you can check inside your `zstor-default.toml` file to see the result. There should be four sections that look like this:
+
+```
+# Node 1
+[[meta.config.back ends]]
+address = "[2a02:1802:5e:0:c11:7dff:fe8e:83bb]:9900"
+namespace = "18-532404-node1meta0"
+password = "Your password"
+```
+
+Sometimes `tfcmd` fails to fetch deployment information from one or more nodes. In that case, you might see one block with blank fields. You can try generating those sections again by changing the script to only target those node IDs.
+
+Here's an example to retry nodes 2 and 3:
+
+```sh
+# Example with nodes 2 and 3. Write out the config sections for the metadata zdbs
+for node in 2 3; do
+  # The rest of the script is the same
+  # ...
+```
+
+Then check the file again. Make sure that all the metadata nodes you specified have a populated config section and that result of any failed attempts are deleted.
+
+### Deploy Data Zdbs
+
+This process is very similar to the deployment of the metadata back ends, with a few small changes to the scripts.
```

Comment:
what is the difference between zdb metadata and zdbs?

Reply:
I've explained a bit more about the difference between each, in terms of the options passed to tfcmd.